### PR TITLE
[TEST] inc coverage of logger service

### PR DESCRIPTION
**Pull request by abzymeinsjtu**
## Purpose of the pull request

increase coverage of logger service

### CODE REVIEW

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/LoggerServiceTest.java [hunk 1]
```diff
@@ -272,9 +312,28 @@ public void testGetLogBytesInSpecifiedProject() {
...
+        taskDefinition.setProjectCode(1L);
+
+        try (
+                MockedStatic<SingletonJdkDynamicRpcClientProxyFactory> ignored =
+                        Mockito.mockStatic(SingletonJdkDynamicRpcClientProxyFactory.class)) {
+            when(SingletonJdkDynamicRpcClientProxyFactory.getProxyClient(any(), any())).thenReturn(iLogService);
+            when(iLogService.pageQueryTaskInstanceLog(any())).thenThrow(ServiceException.class);
+            assertThrowsServiceException(Status.DOWNLOAD_TASK_INSTANCE_LOG_FILE_ERROR,
+                    () -> loggerService.getLogBytes(loginUser, projectCode, 1));
+        }
```
Comment (ruanwenjun) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/LoggerServiceTest.java]: If only adding this kind of bad case is not helpful, can we add a log server provider?

Comment (abzymeinsjtu) [reply dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/LoggerServiceTest.java]: just missed the point, i'll make the change

Comment (abzymeinsjtu) [reply dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/LoggerServiceTest.java]: > If only adding this kind of bad case is not helpful, can we add a log server provider?

updated

**Review by EricGao888 — APPROVED** [review]

**Review by ruanwenjun — APPROVED** [review]
+1

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/utils/ServiceTestUtil.java [hunk 2]
```diff
@@ -27,4 +30,19 @@ public static String randomStringWithLengthN(int n) {
...
+
+    private static User getUser(Integer userId, String userName, UserType userType) {
+        User user = new User();
+        user.setUserType(userType);
+        user.setId(userId);
+        user.setUserName(userName);
+        return user;
+    }
+
+    public static User getAdminUser() {
```
Comment (rickchengx) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/utils/ServiceTestUtil.java:42]: It seems that this part is not used in this PR
Others LGTM

**Pull request merged**